core/build.gradle.kts

```
@@ -27,3 +29,69 @@ dependencies {
     testImplementation(libs.servlet.api)
     testImplementation(libs.slf4j.api)
 }
+
+
+publishing {
+    apply<MavenPublishPlugin>()
+
+    configure<JavaPluginExtension> {
+        withJavadocJar()
+        withSourcesJar()
+    }
+
+    configure<PublishingExtension> {
+        publications {
+            val main by
+            creating(MavenPublication::class) {
+                from(components["java"])
+
+                pom {
+                    name.set("stellar-anchor-sdk")
+                    description.set("Stellar Anchor SDK - Java")
+                    url.set("https://www.github.com/stellar/java-stellar-anchor-sdk")
+                    licenses {
+                        license {
+                            name.set("Apache 2.0")
+                            url.set("https://github.com/stellar/java-stellar-anchor-sdk/blob/main/LICENSE")
+                        }
+                    }
+                    developers {
+                        developer {
+                            id.set("lijamie98")
+                            name.set("Jamie Li")
+                            email.set("jamie@stellar.org")
+                        }
+                        developer {
+                            id.set("JakeUrban")
+                            name.set("Jake Urban")
+                            email.set("jake@stellar.org")
+                        }
+                        developer {
+                            id.set("marcelosalloum")
+                            name.set("Marcelo Salloum")
+                            email.set("jake@stellar.org")
```

👀  my email is wrong:
```suggestion
                            email.set("marcelo@stellar.org")
```

---

Cool catch!!!